core/src/main/python/compare_model.py

```diff
@@ -430,6 +443,8 @@ def _get_change_info(self, current_value, previous_value, location, attribute_na
                 current_text = ','.join(current_list)
                 previous_text = ','.join(previous_list)
                 comment = attribute_name + ": '" + previous_text + "' -> '" + current_text + "'"
+            elif attribute_type == PROPERTIES:
+                comment = property_key + ": '" + str(previous_value) + "'"
             elif not isinstance(previous_value, dict):
                 comment = attribute_name + ": '" + str(previous_value) + "'"
```

Comment:
I think we should mask the password value

Reply:
The password from the new model needs to be exposed in this output, correct?
Are you recommending that we mask the comment with the password from the previous model only?